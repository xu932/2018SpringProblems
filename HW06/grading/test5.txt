name:Copelynn 
name:Anir 
name:Jaxston 
name:Luann 
name:Nataley 
name:Addelynn 
name:Finnick 
name:Aizen 
name:Juandavid 
name:Jayln 
name:Evleen 
name:Samanvi 
name:Amirjon 
name:Sal 
name:Sandy 
name:Srinidhi 
name:Benoit 
name:Pippi 
name:Rayhan 
name:Gayatri 
name:Marianna 
name:Aristotelis 
name:Yeshia 
name:Abdulai 
name:Dedrick 
name:Tessla 
name:Azrael 
name:Chloe 
name:Numa 
