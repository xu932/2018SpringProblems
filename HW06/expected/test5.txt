id:2 
id:4 
id:10 
id:11 
id:12 
id:14 
id:15 
id:16 
id:19 
id:24 
id:27 
id:31 
id:32 
id:34 
id:37 
id:38 
id:40 
id:41 
id:42 
id:43 
id:44 
id:45 
id:46 
id:50 
id:51 
id:53 
id:55 
id:57 
id:62 
id:63 
id:64 
id:66 
id:70 
id:73 
id:74 
id:76 
id:77 
id:79 
id:82 
id:83 
id:86 
id:88 
id:89 
id:90 
id:91 
id:92 
id:93 
id:94 
id:95 
id:96 
id:97 
id:100 
id:101 
id:102 
id:104 
id:105 
id:106 
id:108 
id:110 
id:114 
id:115 
id:117 
id:118 
id:119 
id:120 
id:121 
id:123 
id:124 
id:126 
id:127 
id:128 
id:129 
id:131 
id:134 
id:137 
id:138 
id:140 
id:142 
id:143 
id:145 
id:150 
id:152 
id:153 
id:156 
id:161 
id:162 
id:163 
id:167 
id:170 
id:172 
id:174 
id:177 
id:184 
id:186 
id:192 
id:194 
id:195 
id:196 
id:199 
